app/views/errors/UserInsolventError.scala.html

```diff
@@ -0,0 +1,30 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(mainTemplate: MainTemplate, govukButton: GovukButton)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: config.AppConfig)
+
+@mainTemplate(title = messages("insolventError.title"), appConfig = appConfig) {
+    <h1 class="govuk-heading-xl" id="insolvent-without-access-heading">@messages("insolventError.title")</h1>
```

Comment:
This is just an assumption based on the screenshot, but to me it looks like the design has a smaller heading and a standard paragraph size, rather than the larger ones you've used.

Might be worth asking Bobby to confirm this or provide a link to the page on the prototype if we can't agree based on the screenshot alone.